src/generic/MPoly.jl

```diff
@@ -3348,17 +3348,13 @@ function (a::MPoly{T})(vals::Union{NCRingElem, RingElement}...) where T <: RingE
    powers = [Dict{Int, Any}() for i in 1:length(vals)]
    # First work out types of products
    r = R()
-   c = zero(R)
-   U = Vector{Any}(undef, length(vals))
```

Comment:
huh... what was this vector `U` about?! it was completely unused, wasn't it? 

(Not an objection, I am simply confused)

Reply:
did confuse me as well, but as it was unused I removed it